templates/how-juju-works.html

```diff
@@ -2,8 +2,8 @@
 
 {% block content %}
 <div class="u-fixed-width">
-  <nav class="p-tabs">
-    <ul class="p-tabs__list u-no-margin--top middle-page-tabs" role="tablist">
+  <nav class="p-tabs--25-75">
+    <ul class="p-tabs__list u-no-margin--top" role="tablist">
```

Comment:
It feels `u-no-margin--top` is not really needed? There is no top margin to remove.